nemo/collections/asr/models/rnnt_models.py

```diff
@@ -120,6 +123,32 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self._optim_variational_noise_std = 0
             self._optim_variational_noise_start = 0
 
+        # Setup normalized gradients for model
+        if 'normalize_joint' in self.cfg:
+            self._optim_normalize_joint = self.cfg.normalize_joint
+            self._optim_normalize_txu = None
+        else:
+            self._optim_normalize_joint = False
+            self._optim_normalize_txu = None
+
+        # Setup normalized encoder norm for model
+        if 'normalize_encoder_norm' in self.cfg:
+            self._optim_normalize_encoder_norm = self.cfg.normalize_encoder_norm
+        else:
+            self._optim_normalize_encoder_norm = False
+
+        # Setup normalized decoder norm for model
+        if 'normalize_decoder_norm' in self.cfg:
```

Comment:
Another suggestion is that as all have _norm except one, you may drop that but make the normalize_joint to normalize_joint_txu?

Reply:
Good point. We also need docstrings here.